app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -107,12 +117,12 @@ public void onBindViewHolder(final ViewHolder holder, final int position) {
         holder.progressBarDownload.setProgress(downloadStatus.getDownloadProgress());
 
         if (downloadStatus.isDownloading()) {
-            holder.progressBarDownload.setVisibility(View.VISIBLE);
-            holder.textDownloadProgress.setVisibility(View.VISIBLE);
+            holder.layoutProgress.setVisibility(View.VISIBLE);
+            holder.ivCancelDownload.setVisibility(View.VISIBLE);
```

Comment:
What does `iv` mean?

Reply:
It stands for ImageView.

Reply:
Kan you rename to make it less confusing?